test/interop/python/beginInExportedFunction.chpl

```diff
@@ -0,0 +1,33 @@
+use Time;
+
+// Print "Hello world!" using a begin.
+export proc hello1() {
+  begin writeln('Hello world!');
+  return;
+}
+
+// Test with coforall...
+export proc hello2() {
+  coforall i in 1..4 do writeln('Hello world!');
```

Comment:
Note that coforalls/cobegin/forall don't use dynamic end count (since they are blocking they create their own static end-counts and use them.) I like the tests and don't think we should remove them or anything, just wanted to note they don't use the dynamic end count stuff.

Reply:
Yeah I remember you telling me but thought "ah what the heck". Lydia did suggest making sure to add tests with nested `begin`.